external-crates/move/crates/move-compiler/tests/move_check/liveness/vector_loop.exp

```diff
@@ -0,0 +1,43 @@
+error[E06001]: unused value without 'drop'
+   ┌─ tests/move_check/liveness/vector_loop.move:23:60
+   │
+14 │     struct User has key, store { id: ID }
+   │            ---- To satisfy the constraint, the 'drop' ability would need to be added here
+   ·
+23 │             user_delete(table::remove(&mut table, user_id(&vector::pop_back(users))));
+   │                                                            ^^^^^^^^^^^^^^^^^^^^^^^
+   │                                                            │
+   │                                                            Invalid usage of undroppable value
+   │                                                            The type 'a::m::User' does not have the ability 'drop'
+   │                                                            This term might contain a value without the 'drop' ability and must be used
+
+error[E06001]: unused value without 'drop'
+   ┌─ tests/move_check/liveness/vector_loop.move:25:9
+   │
+14 │     struct User has key, store { id: ID }
+   │            ---- To satisfy the constraint, the 'drop' ability would need to be added here
+   ·
+23 │             user_delete(table::remove(&mut table, user_id(&vector::pop_back(users))));
+   │                                                            -----------------------
+   │                                                            │
+   │                                                            The value is created but not used. The value does not have the 'drop' ability and must be consumed before the function returns
```

Comment:
Nit. We may want to consider unifying terminology here - sometimes we say "use" and sometime "counsume" when talking about values without `drop` ability`